test-alerts.cpp

```diff
@@ -5,4 +5,50 @@
 
 TEST_CASE("infers the breach according to limits") {
   REQUIRE(inferBreach(12, 20, 30) == TOO_LOW);
```

Comment:
Please include boundary conditions too as part of test cases. i.e. lower limit and upper limit value and the expected output.